README.md

```diff
@@ -286,6 +283,7 @@ For more details, see [nginx.org](http://nginx.org/en/docs/).
 * [nginx-lua](https://github.com/fabiocicerchia/nginx-lua) - Nginx 1.19+ with LUA support based on Alpine Linux, Amazon Linux, CentOS, Debian, Fedora and Ubuntu.
 * [nginx-testing](https://github.com/jirutka/nginx-testing) - Support for integration/acceptance testing of nginx configuration in TypeScript/JavaScript.
 * [nginx-binaries](https://github.com/jirutka/nginx-binaries) - Nginx and njs binaries for Linux (x86_64, aarch64, ppc64le), macOS and Windows; Linux binaries are static so works on every Linux.
+* [NPMplus](https://github.com/ZoeyVid/NPMplus)
```

Comment:
Add a description. It would be better. Thanks.

For example: Docker container for managing Nginx proxy hosts with a simple, powerful interface.

Reply:
Did you know github has this feature ... "Add suggestion" button (or ctrl+g):

```suggestion
* [NPMplus](https://github.com/ZoeyVid/NPMplus) - Docker container for managing Nginx proxy hosts with a simple, powerful interface
```